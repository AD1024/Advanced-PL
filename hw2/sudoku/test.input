3
2 0 8 5 3 0 1 0 4
0 5 1 2 4 6 0 9 0
0 6 9 0 0 7 0 3 0
7 2 6 1 9 0 3 4 8
0 0 3 4 0 8 7 0 0
5 8 4 0 2 3 9 1 6
0 1 0 6 0 0 4 8 0
0 3 0 9 7 4 6 5 0
6 0 5 0 8 1 2 0 9