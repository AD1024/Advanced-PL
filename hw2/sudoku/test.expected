3
2 7 8 5 3 9 1 6 4 
3 5 1 2 4 6 8 9 7 
4 6 9 8 1 7 5 3 2 
7 2 6 1 9 5 3 4 8 
1 9 3 4 6 8 7 2 5 
5 8 4 7 2 3 9 1 6 
9 1 7 6 5 2 4 8 3 
8 3 2 9 7 4 6 5 1 
6 4 5 3 8 1 2 7 9 
puzzle solution is unique
